sql/lease.go

```diff
@@ -151,11 +152,13 @@ func (s LeaseStore) Acquire(txn *client.Txn, tableID ID, minVersion DescriptorVe
 		}
 		return nil
 	})
```

Comment:
Ah, yes. I had a TODO here. Changed to `roachpb.NewError(pErr.GoError())`. I think `Acquire` still needs to return `pErr` since this is a nested transaction and `pErr` is eventually passed to `db.Txn()``

```
   LeaseManager.refreshLease()
-> db.Txn() 
-> LeaseManager.acquire()
-> tableState.acquire()
-> tableState.acquireNodeLease()
-> LeaseStore.Acquire()
```